src/_pytest/reports.py

```diff
@@ -368,17 +418,27 @@ def from_item_and_call(cls, item: Item, call: CallInfo[None]) -> TestReport:
                 if excinfo.value._use_item_location:
                     path, line = item.reportinfo()[:2]
                     assert line is not None
-                    longrepr = os.fspath(path), line + 1, r.message
+                    longrepr = (os.fspath(path), line + 1, r.message)
                 else:
                     longrepr = (str(r.path), r.lineno, r.message)
+            elif isinstance(excinfo.value, BaseExceptionGroup):
+                value: BaseExceptionGroup = excinfo.value
+                if value.exceptions and any(
+                    isinstance(exception, skip.Exception)
+                    for exception in value.exceptions
+                ):
+                    outcome = "skipped"
+                    skipped_exceptions = cast(list[Skipped], value.exceptions)
+                    longrepr = _format_exception_group_all_skipped_longrepr(
+                        item, excinfo, skipped_exceptions
+                    )
+                else:
+                    # fall through to your existing failure path
```

Comment:
- For comments, prefer to use proper sentences, i.e. start with capital letter and end with period.
- Who is the "you" in this comment?